### DISCUSSION TITLE
i18n

### DISCUSSION THREAD

**Discussion by paskuale75** [simialbi/yii2-ticket#4] [Q&A]
How can translate with i18n component this:

`$this->title = Yii::t('simialbi/ticket', 'Create ticket');
`

Thanks

**Comment by paskuale75** [comment]
[solved]
I paste here i18n component settings:
```
return [
    'translations' => [
        'app' => [
            'class' => 'yii\i18n\PhpMessageSource',
            'basePath' => '@app/messages',
        ],
        'simialbi/ticket' => [
            'class' => 'yii\i18n\PhpMessageSource',
            'basePath' => '@app/messages',  // this is a default
        ]
    ],
];
```

and in message folder create subfolder /simialbi/ticket ! Good work @simialbi

**Reply by simialbi** [reply]
@paskuale75 Could you send me the italian translation once finished? I'd like to integrate in the system.

**Reply by paskuale75** [reply]
Sure, po format or simple array?

**Reply by simialbi** [reply]
not important. I'll convert it myself.

**Reply by paskuale75** [reply]
@simialbi I've to separate files for each `msgctxt` directive ? I'm writing everything inside the messages.php file

**Reply by simialbi** [reply]
No that is not necessary. Thank you 🙂

**Reply by paskuale75** [reply]
PR maked but Travis check don't pass....

**Reply by simialbi** [reply]
Not important, the checks are not finished yet. Grazie mille
